lib/ecto/schema.ex

```diff
@@ -1143,14 +1200,71 @@ defmodule Ecto.Schema do
       # Update the order
       changeset = Repo.update!(changeset)
 
+  ## Inline embedded schema
+
+  The schema module can be defined inline in the parent schema in simple
+  cases:
+
+      defmodule Parent do
+        use Ecto.Schema
+
+        schema "parents" do
+          field :name, :string
+
+          embeds_one :children, Child do
```

Comment:
Should we change documentation here and use `embeds_many` example in order to avoid confusion?